exporter/views.py

```diff
@@ -11,18 +11,23 @@ def download_export(request):
     spider = request.GET.get("spider")
     job_id = request.GET.get("job_id")
     full = request.GET.get("full")
-    year = request.GET.get("year")
+    year = int(request.GET.get("year"))  # guard against path traversal
```

Comment:
@jpmckinney  This fails is full = True (because "year" is None).  Should we move this to the `if full` `else` block or check here if `request.GET.get("year")` is None/full is True?

Reply:
Yes, let's cast as `int` in the `else` branch.